maplibre-adapter/src/androidTest/java/com/amplifyframework/geo/maplibre/Credentials.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.geo.maplibre
+
+import android.content.Context
+import com.amplifyframework.core.Resources
+
+/**
+ * Utility to load Cognito user credentials from a json document.
+ * The document must be formatted as following:
+ *
+ * <pre>
+ * {
+ *   "username": "username123",
```

Comment:
Question: In the documentation should we add values as "YOUR_AWS_USERNAME" instead of "username123" to remove ambiguity ?

Reply:
The value isn't the AWS username, it's just a username/password for testing purposes that's defined in the integration test resources (no need for anyone running the tests to change the username or password used for these tests).